aten/src/ATen/native/TensorIterator.h

```diff
@@ -126,7 +126,8 @@ struct CAFFE2_API TensorIterator {
   void foreach_reduced_elt(const loop_subiter_t& loop, bool parallelize=true);
 
   static std::unique_ptr<TensorIterator> binary_op(Tensor& out, const Tensor& a, const Tensor& b);
-  static std::unique_ptr<TensorIterator> reduce_op(Tensor& out, const Tensor& a);
+  static std::unique_ptr<TensorIterator> reduce_op(
```

Comment:
can you avoid reformatting code that you don't touch?  It makes it harder to see what's going on for the reviewer.

Reply:
Yes, I will revert it.